docs/topics/gradle.md

```diff
@@ -877,9 +877,16 @@ Each of the options in the following list overrides the ones that came before it
 * If the Gradle daemon's JVM arguments have the `kotlin.daemon.jvm.options` system property – use it in the `gradle.properties` file:
 
  ```properties
-  org.gradle.jvmargs=-Dkotlin.daemon.jvm.options=-Xmx1500m -Xms=500m
+  org.gradle.jvmargs=-Dkotlin.daemon.jvm.options=-Xmx1500m,Xms=500m
   ```
 
+  > In this case, enumerate arguments according to the following rules:
+  > Use the minus sign `-` before the argument `Xmx` and don't use it before all other arguments.
+  > Separate all arguments with the comma ','.
+  > Don't use spaces between arguments, it will lead to the usage of the arguments after a space for the Gradle daemon, not for the Kotlin daemon.
```

Comment:
Isn't this a part of the second rule?
"Separate arguments with commas (`,`) _without spaces_. Arguments that come after a space will be used for .."